src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -128,7 +128,8 @@ public void RunAndBlock(CancellationToken cancellationToken = default(Cancellati
                     // Create a new host config, but keep the host id from existing one
                     _config.HostConfig = new JobHostConfiguration
                     {
-                        HostId = _config.HostConfig.HostId
+                        HostId = _config.HostConfig.HostId,
+                        LoggerFactory = new LoggerFactory()
```

Comment:
What if someone had already set _config.LoggerFactory?

Reply:
There is no property `ScriptHostConfiguration.LoggerFactory` -- that lives in `JobHostConfiguration`, which we construct new here.

BTW -- @fabiocav and I have discussed all these config lifetimes for v2 as it can get pretty confusing what lives from host-to-host and what is created new each time. We want to make sure to address it in Core.

Reply:
Yeah, I meant what if they set _config.HostConfig.LoggerFactory.

Reply:
That being said, I don't think this line is required anyway. I added it while testing something unrelated (which is why the `State = ScriptHostState.Default;` is added below as well). I'll pull them out and test it to make sure.